internal/updater/updater.go

```diff
@@ -28,24 +28,24 @@ import (
 	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
 )
 
-func New(client client.Client) Updater {
-	return Updater{
+func NewBundleUpdater(client client.Client) BundleUpdater {
```

Comment:
I'm a bit partial to the idea of naming this file `updated_bundle.go` as it matches the `updated_bi.go` format. It also makes it obvious what this is an updater of since these functions are not generically applicable. Thoughts?

Reply:
That makes sense, I agree. I'm going to break out the bundle and bundle-instance updaters into their own sub-packages to make this separation even more clear